tiledb/ml/readers/_tensor_schema/ranges.py

```diff
@@ -69,9 +73,14 @@ def __eq__(self, other: Any) -> bool:
         Two ranges are equal if they consist of the same values with the same weights.
         """
 
-    @abstractmethod
     def equal_values(self, other: InclusiveRange[V, W]) -> bool:
         """Check if two ranges consist of the same values."""
+        return (
```

Comment:
At minimal we also need `len(self) == len(other)` because comparing arrays of different size raises `DeprecationWarning` (and will raise an error in the future). The min/max comparisons are not strictly necessary, they're just an optimization (in case they are false).